common/setup_sku_customizations.sh

```diff
@@ -98,20 +95,9 @@ vmSize=\$(echo "\$vmSize" | awk '{print tolower(\$0)}')
 
 ## Topo file setup based on SKU
 case \$vmSize in
-    standard_nc96ads_a100_v4)
-        /opt/azurehpc/customizations/ncv4.sh;;
     
```

Comment:
Any reason we are removing NCv4?

Reply:
This is the AMD GPU section of the file.

Reply:
Correct this is the AMD section so there is no NCv4 AMD GPU SKU